RHEL/7/input/xccdf/system/accounts/physical.xml

```diff
@@ -224,27 +224,38 @@ the non-graphical <tt>multi-user.target</tt> mode.
 </Rule>
 
 <Rule id="disable_interactive_boot" severity="medium">
-<title>Disable Interactive Boot</title>
+<title>Verify that Interactive Boot is Disabled</title>
 <description>
-To disable the ability for users to perform interactive startups,
-edit the file <tt>/etc/sysconfig/init</tt>.
-Add or correct the line:
-<pre>PROMPT=no</pre>
-The <tt>PROMPT</tt> option allows the console user to perform an
-interactive system startup, in which it is possible to select the
-set of services which are started on boot.
+Red Hat Enterprise Linux systems support an "interactive boot" option that can
+be used to prevent services from being started. On Red Hat Enterprise Linux 7
+system interactive boot can be enabled by providing some of <tt>1</tt>,
```

Comment:
Thanks for these nitpicks (since not sure what's the proper form in English)!. Will fix them.